src/main/app/src/app/zaken/intake-afronden-dialog/intake-afronden-dialog.component.html

```diff
@@ -108,17 +118,17 @@
     <mat-expansion-panel-header>
       {{'body' | translate }}
     </mat-expansion-panel-header>
-    <p
+    <div
       class="readonly"
       [innerHTML]="formGroup.get('ontvankelijk').value ? zaakOntvankelijkMail?.body : zaakNietOntvankelijkMail?.body"
-    ></p>
+    ></div>
```

Comment:
Was already in; and for a reason:

The [innerHTML] is therebecause plain HTML is provided in REST response.

Reply:
then it would be 

```
<div *ngIf="formGroup.get('ontvankelijk').value" class="readonly" [innerHTML]="zaakOntvankelijkMail?.body"></div>

<div *ngIf="!formGroup.get('ontvankelijk').value" class="readonly" [innerHTML]="zaakNietOntvankelijkMail?.body"></div>
```

Reply:
fixed ✅